### PR TITLE
downloading_hmi now uses the VSO

### PR DESCRIPTION
**Pull request by nabobalis**
Let us not bother JSOC.

### CODE REVIEW

File: examples/acquiring_data/downloading_hmi.py [hunk 1]
```diff
@@ -12,34 +12,21 @@
...
-# `here <http://jsoc.stanford.edu/ajax/register_email.html>`_.
-
-result = Fido.search(a.Time('2020/01/20 00:00:00', '2020/01/20 00:04:00'),
-                     a.jsoc.Series("hmi.M_720s"),
-                     a.jsoc.Keys(["T_REC, CROTA2"]),
-                     a.jsoc.Notify("jsoc@cadair.com"))
+# To download the required data, we use `sunpy.net.Fido`, a downloader client,
+# to query the Virtual Solar Observatory to acquire HMI data.
+result = Fido.search(a.Time('2020/01/20 00:00:00', '2020/01/20 00:01:00'),
+                     a.Instrument.hmi)
```
Comment (Cadair) [inline examples/acquiring_data/downloading_hmi.py:18]: I think you need to add `a.Physobs.los_magnetic_field` to get the same two results?

Comment (nabobalis) [reply examples/acquiring_data/downloading_hmi.py:18]: Good point.

File: examples/acquiring_data/downloading_hmi.py [hunk 2]
```diff
@@ -12,34 +12,20 @@
...
 ###############################################################################
 # The following shows how to download the results. If we
 # don't provide a path it will download the file into the sunpy data directory.
 # The output provides the path of the downloaded files.
-# The result can be from several data clients, so we have to index the client first
-# and then index the file.
-
-# Slice the first record returned by the first client.
-downloaded_file = Fido.fetch(result[0, 0])
+downloaded_file = Fido.fetch(result)
```
Comment (Cadair) [inline examples/acquiring_data/downloading_hmi.py:28]: I would leave this, we are now printing two file names where as before we showed how to only print one?

Comment (nabobalis) [reply examples/acquiring_data/downloading_hmi.py:28]: There is only one file in the result.

Comment (Cadair) [reply examples/acquiring_data/downloading_hmi.py:28]: huh ok, why does JSOC return an extra file?

Comment (nabobalis) [reply examples/acquiring_data/downloading_hmi.py:28]: Not a clue.

Comment (nabobalis) [reply examples/acquiring_data/downloading_hmi.py:28]: Current download:
```python
Results from 1 Provider:

1 Results from the VSOClient:
   Start Time [1]       End Time [1]    Source Instrument   Type    Wavelength [2]
                                                                       Angstrom
------------------- ------------------- ------ ---------- -------- ----------------
2020-01-20 00:00:22 2020-01-20 00:00:23    SDO        HMI FULLDISK 6173.0 .. 6174.0


Files Downloaded: 100%|█████████████████████████████████████████████████████████████████| 1/1 [00:01<00:00,  1.91s/file]
['/home/nabil/sunpy/data/hmi_m_45s_2020_01_20_00_01_30_tai_magnetogram.fits']
```

**Review by Cadair — APPROVED** [review]

**Pull request merged**